variables.tf

```
@@ -16,21 +16,24 @@ variable "deploy_demo" {
 }
 
 variable "clusters" {
-  description = "K3s cluster definition"
+  description = "Cluster definition"
```

because `custom_rancher_password` is included below:
```suggestion
  description = "Cluster definition"
  sensitive   = true
```